src/hugo/content/en-us/implementations/aws/command/command2/_index.md

```diff
@@ -0,0 +1,721 @@
+
+    except Exception as e:
+        exit(e)
+
+    # Wait timer to receive delta events when device remain connected after initial connect/reconnect   
+    try:
+        while True:
+            time.sleep(1)
+    except KeyboardInterrupt:
+        # Disconnect
+        print("Disconnecting...")
+        disconnect_future = mqtt_connection.disconnect()
+        disconnect_future.result()
+        print("Disconnected!")
+
+```
+- Update shadow request document 
+```json
+
+{
+  "state": {
+    "desired": {
+      "deviceAttrib" :{
+      		"motorStatus" : "on"
+      },
+      "command":[{
+      		"relay-command":{
+                "message": "Message sent as part of command action",
+      		    "topic":"cmd/device1/response",
+      		    "ttl":300
+              }
+      }]
+    },
+    "reported": {
+      "deviceData" :{
+      		"motorStatus" : "off"
+      }
+    }
+  },
+  "clientToken":"myapp123"
+}
+
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Application
+
+The _Client Application_ code connects to the Broker and subscribes to the _Device_'s response topic, as well as reserved MQTT topics for device shadow. The _Client Application_ can process device reported state using reserved shadow topics. Reference implementation for complete processing on the _Client Application_ can be tweaked using device implementation above.
+
+Below implementation uses AWS CLI as a quick demonstration on how an application can interact with a shadow using command line.
+
+- To get the current state of the shadow using the AWS CLI. From the command line, enter this command.
+
+```cmd
+
+aws iot-data get-thing-shadow --thing-name REPLACE_WITH_DEVICE_NAME  /dev/stdout
+
+```
+
+- To update the shadow from command line using AWS CLI
+
+```cmd
+
+
+aws iot-data update-thing-shadow --thing-name REPACE_WITH_DEVICE_NAME \
+    --cli-binary-format raw-in-base64-out \
+    --payload '{"state":{"desired":{"deviceAttrib":{"motorStatus":"on"},"command":[{"relay-command":{"message":"Message to relay back","topic":"relay-command/response","ttl":300}}]}},"clientToken":"myapp123"}' /dev/stdout
+
+```
+
+
+## Considerations
+
+This implementation covers the basics of a command-and-control pattern. It does not cover certain aspects that may arise in production use.
+
+#### Choosing to use named or unnamed shadows
+The Device Shadow service supports named and unnamed, classic shadows, as have been used in the past. A thing object can have multiple named shadows, and no more than one unnamed, classic shadow. A thing object can have both named and unnamed shadows at the same time; however, the API used to access each is slightly different, so it might be more efficient to decide which type of shadow would work best for your solution and use that type only. For more information about the API to access the shadows, see [Shadow topics](https://docs.aws.amazon.com/iot/latest/developerguide/reserved-topics.html#reserved-topics-shadow).
+
+#### Message Versioning and Order
```

Comment:
You might want to link or reference to where the version number can be found and used.